spring-security-core/.gitignore

```diff
@@ -1,13 +1,2 @@
-*.class
-
 #folders#
-/target
-/neoDb*
-/data
-/src/main/webapp/WEB-INF/classes
-*/META-INF/*
-
```

Comment:
intended? just making sure.

Reply:
Yes, a lot of things that were ignored could never occur.